scrapinghub/client/utils.py

```diff
@@ -106,6 +107,20 @@ def __init__(self, cls, client, key):
             self._proxy_methods(methods)
             self._wrap_iter_methods([method[0] for method in methods])
 
+        if issubclass(cls, MappingResourceType):
+            methods = ['ignore_fields', 'expire', 'save', 'liveget',
```

Comment:
After some tests I think we shouldn't proxy `_cached` method here: looks like it was mainly used when instantiating JobMeta with some pre-cached data, but now JobMeta constructor is not public. We still could add it as a property to set it after instantiating jobmeta, but I'm not sure that it's actually used by anyone